src/matlab2tikz.m

```diff
@@ -1519,33 +1539,29 @@ case guitypes()
     % a reasonable default.
     matlabDefaultNumMinorTicks = 3;
     if ~isempty(ticks)
-        options = opts_add(options, ...
-            [axis,'tick'], sprintf('{%s}', ticks));
+        options = opts_add(options, [axis,'tick'], sprintf('{%s}', ticks));
```

Comment:
There is no need to break those lines, so I rather have everything on the same line if we can since it is slightly better for readability IMHO.